scripts/zones/Bastok_Markets_[S]/npcs/Kevan.lua

```diff
@@ -0,0 +1,45 @@
+-----------------------------------
+-- Area: Bastok Markets [S]
+--  NPC: Kevan
+-- !pos -308.590 -012.000 -094.227 189
+-----------------------------------
+local ID = require("scripts/zones/Bastok_Markets_[S]/IDs")
+require("scripts/globals/keyitems")
+require("scripts/globals/settings")
```

Comment:
We don't use keyitems nor settings requires anymore.

Reply:
Oh cool!  These were necessary before but I didn't think to try without, thanks!